src/tingle.js

```diff
@@ -365,14 +467,19 @@
 
         this.modal.addEventListener('mousedown', this._events.clickOverlay);
         window.addEventListener('resize', this._events.resize);
-        document.addEventListener("keydown", this._events.keyboardNav);
+        document.addEventListener("keyup", this._events.keyboardNav);
     }
 
     function _handleKeyboardNav(event) {
         // escape key
         if (this.opts.closeMethods.indexOf('escape') !== -1 && event.which === 27 && this.isOpen()) {
             this.close();
         }
+
+        // tab key
+        if (event.which === 9 && this.isOpen()) {
```

Comment:
What is that?

Reply:
Well I don't know :o

I suppose I forgot to remove this test :)